packages/terra-dialog-modal/package.json

```diff
@@ -37,7 +37,7 @@
     "terra-action-header": "^2.0.0",
     "terra-button": "^3.3.0",
     "terra-date-picker": "^4.48.0",
-    "terra-form-select": "^6.8.0",
+    "terra-form-select": "file:./terra-form-select.tgz",
```

Comment:
nope

Reply:
Yeah this I will revert once the form-select [PR](https://github.com/cerner/terra-core/pull/3167) is merged.

Reply:
@saket2403 PR is now merged and released. :)

Reply:
Removed Dependency